programs/cleos/main.cpp

```diff
@@ -1693,6 +1703,27 @@ struct claimrewards_subcommand {
    }
 };
 
+struct setproxylvl_subcommand {
+   string account;
+   string symbol;
+   int8_t level;
+
+   setproxylvl_subcommand(CLI::App* actionRoot) {
+      auto set_proxy_level_action = actionRoot->add_subcommand("setproxylvl", localized("Set an account proxy level"));
+      set_proxy_level_action->add_option("account", account, localized("An account whose proxy level will be set"))->required();
+      set_proxy_level_action->add_option("symbol", symbol, localized("A symbol of an asset used in the system"))->required();
+      set_proxy_level_action->add_option("level", level, localized("A proxy level to set"))->required();
+
+      set_proxy_level_action->set_callback([this] {
+          const auto proxy_status = call(get_proxy_status_func, fc::mutable_variant_object("account", account)("symbol", symbol));
+
+          if (proxy_status["proxylevel"].as_uint64() != level) {
```

Comment:
may be it is better to explicitly get "proxy level has not been changed" assert from cyber.stake in such case, what do you think?

Reply:
or just get an explicit warning